contrib/inventory/netbox.py

```diff
@@ -0,0 +1,378 @@
+#!/usr/bin/env python
+
+# (c) 2017, Ahmed AbouZaid <http://aabouzaid.com/>
+#
```

Comment:
Could you please replace line 4 to 18 with following 

```
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Reply:
Fixed.